### PR TITLE
feat: add seed phrase to backup-keys

### PR DESCRIPTION
**Pull request by bashbunni**
## Changes:
- [x] add melt seed phrase output for backups
- [x] make melt seed phrase look :rabbit: cute :rainbow: (UI changes)
ui changes

## Related Issues:
#116

### CODE REVIEW

**Comment by bashbunni** [comment]
I copied a lot of code over from melt :joy: let me know if there's a nicer way I could organize things or if it's okay

**Comment by caarlos0** [comment]
> I copied a lot of code over from melt 😂 let me know if there's a nicer way I could organize things or if it's okay

I noticed hehe

I was thinking we can refactor there a bit to make it easier to use these parts as a lib too... will take a look there and ping you once its done :)

File: cmd/backup_keys.go [hunk 1]
```diff
@@ -174,3 +202,84 @@ func createTar(source string, target string) error {
...
+	if err != nil {
+		pwderr := &ssh.PassphraseMissingError{}
+		if errors.As(err, &pwderr) {
+			pass, err := askKeyPassphrase(path)
+			if err != nil {
+				return "", err
+			}
+			return backup(path, pass)
+		}
+		return "", fmt.Errorf("could not parse key: %w", err)
```
Comment (moisesvega) [inline cmd/backup_keys.go:222]:
```suggestion
		if !errors.Is(err, &ssh.PassphraseMissingError{}) {
			return "", fmt.Errorf("could not parse key: %w", err)
		}

		pass, err := askKeyPassphrase(path)
		if err != nil {
			return "", err
		}
		return backup(path, pass)
```

File: cmd/backup_keys.go [hunk 2]
```diff
@@ -174,3 +202,84 @@ func createTar(source string, target string) error {
...
+func askKeyPassphrase(path string) ([]byte, error) {
+	defer fmt.Fprintf(os.Stderr, "\n")
+	return readPassword(fmt.Sprintf("Enter the passphrase to unlock %q: ", path))
+}
+
+func readPassword(msg string) ([]byte, error) {
+	fmt.Fprint(os.Stderr, msg)
+	t, err := tty.Open()
+	if err != nil {
+		return nil, fmt.Errorf("could not open tty")
```
[the remark this answers is not in the document]
Comment (caarlos0) [reply cmd/backup_keys.go:243]: its actually missing the parent err:

```suggestion
		return nil, fmt.Errorf("could not open tty: %w", err)
```

**Comment by caarlos0** [comment]
alternative idea to this: an option to print the key to stdout, so users can pipe into melt (or something else) 🤔

**Pull request closed without merging**